core/remote/watcher.js

```diff
@@ -82,6 +82,7 @@ class RemoteWatcher {
   async watch () {
     try {
       const seq = await this.pouch.getRemoteSeqAsync()
+      this.events.emit('online')
```

Comment:
I guess this would unpause the GUI but not the Sync. User would see the app as unpaused, but pending Sync changes would still not happen (looks like the behavior I got during validation).

Reply:
(not a blocker since Sync should have unpaused itself after 1 minute)